shell/platform/fuchsia/dart-pkg/zircon/sdk_ext.gni

```diff
@@ -0,0 +1,12 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+zircon_sdk_ext_lib = "//topaz/public/dart-pkg/zircon/lib/zircon.dart"
+
+zircon_sdk_ext_files = [
+  zircon_sdk_ext_lib,
+  "//topaz/public/dart-pkg/zircon/lib/src/handle.dart",
```

Comment:
are these right?

Reply:
I dont think this file is being referenced. I'll get rid of it.